src/troute_model.py

```diff
@@ -101,10 +101,15 @@ def preprocess_static_vars(self, values: dict):
             from_files=False, value_dict=values,
             bmi_parameters=self._bmi_parameters,)
 
-        values['diffusive_tw_ids'] = self._network._nexus_latlon.id.values
-        values['diffusive_tw_latitude'] = self._network._nexus_latlon.lat.values
-        values['diffusive_tw_longitude'] = self._network._nexus_latlon.lon.values
-        
+        if (len(self._network._nexus_latlon) > 0):
+            values['diffusive_tw_ids'] = self._network._nexus_latlon.id.values
+            values['diffusive_tw_latitude'] = self._network._nexus_latlon.lat.values
+            values['diffusive_tw_longitude'] = self._network._nexus_latlon.lon.values
+        else:
+            values['diffusive_tw_ids'] = pd.DataFrame()
+            values['diffusive_tw_latitude'] = pd.DataFrame()
+            values['diffusive_tw_longitude'] = pd.DataFrame()          
```

Comment:
Could you makes these np.arrays instead of empty dataframes? We need 1D arrays for BMI variables.

Reply:
OK - done.